src/Components/Shared/Inputs/index.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import styles from './input.module.css';
+
+export function Input({ labelText, value, type, name, change, placeholder, blur }) {
```

Comment:
I think you should add a "for" attribute for the label, it's important for accesibility!